### DISCUSSION TITLE
Column "id" does not exist

### DISCUSSION THREAD

**Discussion by joshpachner** [supabase/supabase#15628] [Questions]
Hello, I just need validation from the community to tell me im not crazy...

Using the Supabase UI to insert row into a table gives me this error. Which doesn't make sense... since the column "id" clearly exists... 
I also tried using normal sql "insert into ... " and still gave me that error. 

Anyone else encountered this? Can someone shine some light on it for me?

<img width="519" alt="image" src="https://github.com/supabase/supabase/assets/65133879/afcc2472-1c0c-42b8-9ac2-cac586b790f2">

**Comment by GaryAustin1** [comment]
Does seem odd.  I'd try and hard refresh the browser and restart the database if not production as a first step.

**Reply by joshpachner** [reply]
I ended up not restarting the database. We are using Prisma as our orm, so i just made changes, pushed the new table, and then made changes back, and then pushed again. And now there is no problem with ID not existing.

So might have been a Prisma issue... ?
